srcpkgs/syncthing/template

```diff
@@ -42,5 +43,23 @@ syncthing-relaysrv_package() {
 		vmove usr/bin/strelaysrv
 		vlicense cmd/strelaysrv/LICENSE
 		vsv relaysrv
+		vman man/strelaysrv.1
+	}
+}
+
+syncthing-discosrv_package() {
+	short_desc+=" - discovery server"
+	system_accounts="discosrv"
```

Comment:
can we get this to `_discosrv` instead, `relaysrv` uses the old way because changing it is annoying.

Reply:
Did that and also fixed the run-script.